### ISSUE TITLE
A11y_VisualStudioCodeClient_Command Palette_UseOfColor: There is no other means provided except color to differentiate the current line and line to be inserted.

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#141720]
### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Visual Studio Code Client; #BM-VisualStudioCode-Win32-Nov2021; #Win32; #UseOfColor #WCAG1.4.1; #AINotInScope; #FTP; #STP;

### Environment Details:
Application Name: Visual Studio Code Client
Windows Version: Windows 11

### Repro Steps:
1) Open Visual studio code editor  
2) Press ctrl+g or Open Go to Symbol in WorkSpace from keyboard shortcut
3)Verify whether there is any other means provided except color to differentiate the current line and line to be inserted.

### Actual Result:
 There is no other means provided except color to differentiate the current line and line to be inserted.

### Expected Result:
 There is should be other means provided except color to differentiate the current line and line to be inserted.

### User Impact:
Color blind Users will face difficulty to figure out which line in code highlighted and the purpose of it.

**Comment by isidorn** [comment]
You can look at the quick pick which renders the line number.
So that differentiates it apart from the colour. Thus closing as invalid
![Screenshot 2022-01-28 at 11 46 26](https://user-images.githubusercontent.com/1926584/151533761-7cfe26a3-8855-4047-a6eb-e7e119716a6f.png)
